conanfile.py

```diff
@@ -27,31 +34,20 @@ class LlfsConan(ConanFile):
     default_options = {"shared": False}
     build_policy = "missing"
 
+    exports = [
+        "script/*.py",
+        "script/*.sh",
+    ]
     exports_sources = [
         "src/CMakeLists.txt",
         "src/**/*.hpp",
         "src/**/*.ipp",
         "src/**/*.cpp",
-        "script/*.py",
-        "script/*.sh",
     ]
-
-    #+++++++++++-+-+--+----- --- -- -  -  -   -
-    def _append_script_dir(self):
-        script_dir1 = os.path.join(os.path.dirname(__file__), 'script')
-        sys.path.append(script_dir1)
-
-        try:
-            script_dir2 = os.path.join(os.path.split(self.source_folder)[0], 'script')
-            sys.path.append(script_dir2)
-        except:
-            pass
+            
     #+++++++++++-+-+--+----- --- -- -  -  -   -
 
     def set_version(self):
-        self._append_script_dir()
-        import batt
-        batt.VERBOSE = VERBOSE
```

Comment:
Do we want to leave VERBOSE on?

Reply:
We can just let batt.py set VERBOSE independently.  It uses the same method.